clang/test/CodeGenCXX/wasm-em-eh.cpp

```diff
@@ -0,0 +1,13 @@
+// RUN: %clang_cc1 %s -triple wasm32-unknown-unknown -fexceptions -fcxx-exceptions -emit-llvm -o - -std=c++11 2>&1 | FileCheck %s
```

Comment:
should this triple be wasm32-unknown-emscripten? Or do we use em-exceptions by default on all wasm triples?

Reply:
We don't do anything differently wrt EH depending on whether it is `unknown` or `emscripten` triple. But given that this test only applies to Emscripten, I changed it to `wasm-unknown-emscripten`.